torch/quantization/quantize.py

```diff
@@ -32,32 +33,22 @@ def _propagate_qconfig_helper(module, qconfig_dict, skip_list=None,
     Return:
         None, module is modified inplace with qconfig attached
     """
-    if skip_list is None:
-        skip_list = DEFAULT_SKIP_LIST
-    if type(module) in skip_list:
-        module.qconfig = None
-    if not hasattr(module, 'qconfig'):
-        module.qconfig = qconfig_parent
-        if qconfig_dict:
-            if prefix in qconfig_dict:
-                module.qconfig = qconfig_dict[prefix]
-            elif type(module) in qconfig_dict:
-                module.qconfig = qconfig_dict[type(module)]
-
-    # Don't quantize empty Sequential, empty Sequential is same as
-    # Identity, but we can't put Sequential into skip list because
-    # we also have non-empty Sequential and the qconfig needs to
-    # be propagated to child in that case
     # TODO: Add test
-    if len(module._modules) == 0 and type(module) == nn.Sequential:
-        module.qconfig = None
+    if white_list is None:
+        white_list = DEFAULT_QCONFIG_PROPAGATE_WHITE_LIST
 
+    module_qconfig = qconfig_dict.get(type(module), qconfig_parent)
+    module_qconfig = qconfig_dict.get(prefix, module_qconfig)
+    module_qconfig = getattr(module, 'qconfig', module_qconfig)
+
+    if type(module) in white_list:
+        module.qconfig = module_qconfig
     for name, child in module.named_children():
         module_prefix = prefix + '.' + name if prefix else name
-        _propagate_qconfig_helper(child, qconfig_dict, skip_list,
-                                  module.qconfig, module_prefix)
+        _propagate_qconfig_helper(child, qconfig_dict, white_list,
+                                  module_qconfig, module_prefix)
 
-# TODO(jerryzh): expose skip_list
+# TODO(jerryzh): expose white_list
 def propagate_qconfig_(module, qconfig_dict=None):
```

Comment:
We should expose the whitelist as part of the top-level API (prepare/prepare_qat/quantize/quantize_qat).

Reply:
There is a TODO that jerry left. It might be part of a different PR.